ydb/core/protos/auth.proto

```diff
@@ -49,6 +49,7 @@ message TAuthConfig {
     optional bool UseAccessServiceApiKey = 76 [default = false]; // Use IAM ApiKey
     optional string AsSignatureExpireTime = 77 [default = "1m"];
     optional bool UseBuiltinDomain = 78 [default = true];
+    optional string AccessServiceType = 79 [default = "Yandex"]; // For now the following values are supported: "Yandex", "Nebius"
```

Comment:
maybe it's better to add a number as a version? just in case?

Reply:
Thank you! It's a good idea